proto/cosmos/staking/v1beta1/query.proto

```diff
@@ -90,6 +91,34 @@ service Query {
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/cosmos/staking/v1beta1/params";
   }
+
```

Comment:
Do these not need `(google.api.http).get` options set?

Reply:
added [[here](https://github.com/cosmos/cosmos-sdk/pull/16747/commits/7aadb58e34a410abb8864a520efd678bf84fbd11)]